tests/dut_console/test_idle_timeout.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import pexpect
+import logging
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_TMOUT = "900"
+SET_TMOUT = "10"
+
+def test_timeout(duthost_console, duthost):
+    logger.info("Get default session idle timeout")
+    default_tmout = duthost_console.send_command('echo $TMOUT')
+    pytest_assert(default_tmout == DEFAULT_TMOUT, "default timeout on dut is not 900 seconds")
```

Comment:
It would be better to use:
`
pytest_assert(default_tmout == DEFAULT_TMOUT, "default timeout on dut is not {} seconds".format(DEFAULT_TMOUT))
`
If we change DEFAULT_TMOUT in the future, the failure message will always be clear.

Reply:
I will modify this.